packages/server/lib/cross-origin/process-callback.ts

```diff
@@ -0,0 +1,67 @@
+import { getFullWebpackOptions } from '@cypress/webpack-batteries-included-preprocessor'
+import md5 from 'md5'
+import { fs } from 'memfs'
+import * as path from 'path'
+import webpack from 'webpack'
+
+const VirtualModulesPlugin = require('webpack-virtual-modules')
+
+interface Options {
+  file: string
+  fn: string
+}
+
+// @ts-ignore - webpack expects `fs.join` to exist for some reason
```

Comment:
```suggestion
// @ts-expect-error - webpack expects `fs.join` to exist for some reason
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/25626/commits/b5270d5fad626668014d648d7199674cac7e3d0c